run_tests.py

```
@@ -0,0 +1,189 @@
+    
+    # Check if we're in the right directory
+    if not Path("agents").exists() or not Path("backend").exists():
+        print("❌ Please run from the project root directory")
+        return False
+    
+    print("✅ Project structure validated")
+    
+    # Try importing key modules
+    try:
+        sys.path.append(str(Path.cwd()))
+        from agents.config import AgentConfig
```

Ruff (0.11.9)

`agents.config.AgentConfig` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)